src/pages/paymentInfo/paymentinfo.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { PaymentInfoPage} from './paymentinfo';
+import { IonicPageModule } from 'ionic-angular';
+
+@NgModule({
+  declarations: [PaymentInfoPage],
+  imports: [IonicPageModule.forChild(PaymentInfoPage)],
+  entryComponents: [
+    PaymentInfoPage
+  ]
+})
+export class PaymentInfoPageModule { }
```

Comment:
This file also needs to be name `paymentInfo.module.ts`

Reply:
changed the paymentinfoPage error.

Reply:
@kevinburnside the file `paymentinfo.module.ts` still needs to be renamed to`paymentInfo.module.ts`